### PR TITLE
Gluten Free option to Master/Origin

### PR DESCRIPTION
**Pull request by jackmasterson**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2486)

### CODE REVIEW

File: english/gluten-free.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Marshmallows are gluten free! They are also probably calorie free
```
Comment (casey-collab) [inline english/gluten-free.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline english/gluten-free.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

File: english/gluten-free.md (hunk 1 above)
Comment (casey-collab) [inline english/gluten-free.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/gluten/gluten-free.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Marshmallows are gluten free! They are also probably calorie free
+
+and are nutritiously delicious! Eat some after every workout and one day you'll
+
+become [Batman](I'm-batman/batman.md)
```
Comment (casey-collab) [inline english/gluten/gluten-free.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/gluten/gluten-free.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+Marshmallows are gluten free! They are also probably calorie free
```
Comment (casey-collab) [inline english/gluten/gluten-free.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/gluten-free.md (hunk 1 above)
Comment (casey-collab) [inline english/gluten-free.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**